driver/aliases.py

```diff
@@ -7,18 +7,18 @@
 
 ALIASES = {}
 
-
 ALIASES["seq-sat-fd-autotune-1"] = [
-    "--evaluator", "hff=ff(transform=adapt_costs(one))",
-    "--evaluator", "hcea=cea()",
-    "--evaluator", "hcg=cg(transform=adapt_costs(plusone))",
-    "--evaluator", "hgc=goalcount()",
-    "--evaluator", "hAdd=add()",
-    "--search", """iterated([
+    "--search",
+    "let(hff, ff(transform=adapt_costs(ONE)),"
```

Comment:
previously enums were case insensitive, now there are sensitive.We have to decide what we ant and implement this.

Reply:
I think there is no reason to change from case insensitive to case sensitive as part of this issue. If we want this change it should be a separate issue.
With case insensitive enums, I would still treat lower case as the "canonical" representation, so this should be lower case here.

Reply:
... and in the other aliases as well.